lib/view/templates.py

```diff
@@ -69,24 +69,44 @@ def project_xdr_req_shipped_errors(s, esc, ess):
     return esc + ess
 
 
+def _ignore_zero(num: int):
+    if num == 0:
+        return None
+
+    return num
+
+
+def _ignore_null(s: str):
+    if s == "null":
+        return None
+
+    return s
+
+
 #
 # Aggregator helpers
 #
 
 
 def weighted_avg(weights: Iterable[float], values: Iterable[float]):
-    total = 0.0
-    weighted_avg = 0.0
+    """
+    Computes the average of multiple percentage points. Remember: used/total = percent or percent * total = used
+    Let's assume each entry has three pieces of info (used amount, total amount available, percent). To compute
+    the average of percents we can use sum(used for each element) / sum(total for each element) or because
+    percent * total = used we can do sum(percent * total for each element) / sum(total for each element).
+    """
+    values_total = 0.0
+    weighted_total = 0.0
 
     for w, v in zip(weights, values):
-        weighted_avg += w * v
-        total += v
+        weighted_value = w * v
+        weighted_total += weighted_value
+        values_total += v
```

Comment:
I thought weighted averages divide by the sum of the weights in the end. Please double check that this is correct.

Reply:
renamed the variables to match the equation you saw

Reply:
That's what happens when you derive the equation yourself haha.  I was thinking the pct should be considered the weight and the bytes should be considered the values but in more abstract terms the values should be the same unit of the result you are expecting